src/modules/mentorships/interfaces/mentorship.interface.ts

```diff
@@ -0,0 +1,23 @@
+import { Document, ObjectID } from 'mongoose';
+
+export enum Status {
+  NEW = 'New',
+  VIEWED = 'Viewed',
+  APPROVED = 'Approved',
+  REJECTED = 'Rejected',
+  TERMINATED = 'Terminated',
```

Comment:
I believe we should add `canceled` if the mentee already found a mentor or changed their mind.